clients/reality-data/src/RealityDataClient.ts

```diff
@@ -16,13 +16,20 @@ import { WsgClient } from "./wsg/WsgClient";
 /** Currenlty supported  ProjectWise ContextShare reality data types
  * @internal
  */
-export enum RealityDataType {
-  REALITYMESH3DTILES = "RealityMesh3DTiles", // Web Ready Scalable Mesh
-  OPC = "OPC", // Orbit Point Cloud
-  TERRAIN3DTILE = "Terrain3DTiles", // Terrain3DTiles
-  OMR = "OMR", // Mapping Resource,
-  CESIUM_3DTILE = "Cesium3DTiles" // Cesium 3dTiles
-}
+export type RealityDataType =
+  "RealityMesh3DTiles" | // Web Ready Scalable Mesh
+  "OPC" | // Orbit Point Cloud
+  "Terrain3DTiles" | // Terrain3DTiles
+  "OMR" | // Mapping Resource,
+  "Cesium3DTiles"; // Cesium 3dTiles
+
+// TEMPORARY FIX: Continued support of type filtering
```

Comment:
I'm not suggesting you do this, but you *could* eliminate the redundancy like this:
```
const supportedTypes = [
  "RealityMesh3DTiles",
  "OPC",
  "Terrain3DTiles",
  "OMR",
  "Cesium3DTiles",
] as const;

type RealityDataType = typeof supportedTypes[number];
```
I'd only do that if you intend for RealityDataType to remain `@internal`; otherwise the documentation would look very confusing.